public/css/main.css

```
@@ -230,3 +230,11 @@ tr {
 .status-color-dot.none {
   background-color: #2ecc71;
 }
+
+#libLink {
+    text-decoration: none;
+}
+
+#libLink span:hover {
```

```suggestion
#libLink:hover span {
```
Listen to hover on the `a` element, not the span.